### DISCUSSION TITLE
Error in Solving Static Linear Elastic Problem

### DISCUSSION THREAD

**Discussion by anoopngmys** [OpenRadioss/OpenRadioss#1693] [Users]
Hi, I am new to OpenRadioss and trying to solve static linear elastic problem (see below pic.)
![image](https://github.com/OpenRadioss/OpenRadioss/assets/130025938/07693a2c-690f-4b62-8b0f-f7f77f6a2b8b)

I am unable to get solution i.e., H3D file, Anim file, Time history file (not sure, if it prints for static problem) and getting an error in engine.out file (see below pic.)

![image](https://github.com/OpenRadioss/OpenRadioss/assets/130025938/3f277713-8f51-4051-9300-aea85f7f09b1)

Below I am attaching the .rad, .out and other files.
[Test_case1.zip](https://github.com/OpenRadioss/OpenRadioss/files/12011209/Test_case1.zip)

Please help me in resolving the issue. 

Thanks.

**Comment by PaulAltair** [comment]
The MUMPS (implicit) solvers are not included/linked in the OpenRadioss builds by default, you may add them (there are other threads for this: see: https://github.com/orgs/OpenRadioss/discussions/485) but running your input, you should get message stating that: I think erroring out because you tried to run mpi version possibly?
![image](https://github.com/OpenRadioss/OpenRadioss/assets/98418895/d97cc944-accb-4a33-97b6-2103527aac24)

**Reply by anoopngmys** [reply]
Hey Paul,

Thank you for the feedback.

I want to use Preconditioned Conjugate Gradient (PCG) solver not  the Direct solver MUMPS. In that case do I need to follow the steps mentioned in link you provided?

**Reply by PaulAltair** [reply]
Yes, you were requesting Mumps, not PCG, (your /IMPL/SOLVER Solver type was set 0, default = 2) but switching, I still get the Mumps required message, once compiled for Mumps it works though: see attached
[IMPL_run.zip](https://github.com/OpenRadioss/OpenRadioss/files/12052051/IMPL_run.zip)
